src/Core/Localization/CLDR/Currency.php

```diff
@@ -140,6 +140,10 @@ public function getDecimalDigits()
      */
     public function getDisplayName($countContext = CurrencyInterface::DISPLAY_NAME_COUNT_DEFAULT)
     {
+        if (empty($this->displayNames)) {
+            return $this->isoCode;
```

Comment:
Could you add an Unit Test in `tests/Unit/Core/Localization/CLDR/CurrencyTest.php` ?

Reply:
Yep good call!

Reply:
Ping me after that and I will approve a new time :heart_eyes_cat: